WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -126,4 +128,23 @@ public Money CalculateMinReasonableOutputAmount()
 							ScriptType.Taproot.EstimateInputVsize() + ScriptType.Taproot.EstimateOutputVsize()));
 		return Math.Max(minEconomicalOutput, AllowedOutputAmounts.Min);
 	}
+
+	/// <returns>Smallest effective denom that's larger than min reasonable output amount. </returns>
+	public Money CalculateSmallestReasonableEffectiveDenomination()
```

Comment:
This function is not deterministic, what means if you call it twice it can return different numbers.

Reply:
What makes this method not deterministic? The random pick between the Segwit v0 and Taproot output script type?

Reply:
It's deterministic now.